app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,102 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class TaskServiceTest {
+
+    // 1. list -> getTasks
+    // 2. detail -> getTask (with Id)
+    // 3. create -> createTask (with source)
+    // 4. update -> updateTask (with Id, source)
+    // 5. delete -> deleteTask (with Id)
+
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "!!!";
+
+    @BeforeEach
+    void setUp() {
+        // subject
+        taskService = new TaskService();
+
+        // fixtures
+        Task task = new Task();
+        task.setTitle(TASK_TITLE);
+
+        taskService.createTask(task);
+    }
+
+    @Test
+    void getTasks() {
+        List<Task> tasks = taskService.getTasks();
+
+        assertThat(tasks).hasSize(1);
+
+        Task task = tasks.get(0);
+        assertThat(task.getTitle()).isEqualTo(TASK_TITLE);
+    }
```

Comment:
`task`변수를 만드는 것보다 인라인 변수를 하는게 더 괜찮을 것 같아요
```suggestion
    @Test
    void getTasks() {
        List<Task> tasks = taskService.getTasks();

        assertThat(tasks).hasSize(1);
        assertThat(tasks.get(0).getTitle()).isEqualTo(TASK_TITLE);
    }
```

Reply:
네 인라인 변수가 나은것 같습니다 :) 수정했습니다!